build/package/.atlascli.goreleaser.yml

```diff
@@ -39,8 +39,8 @@ builds:
             - VERSION={{ .Version }}
           output: true
 archives:
-- id: linux
-  name_template: "mongodb-atlas-cli_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
+- id: linux_archives
+  name_template: mongodb-atlas_{{ .Version }}_{{ .Os }}_{{- if eq .Arch "amd64" }}x86_64{{- else if eq .Arch "darwin" }}macos{{- else }}{{ .Arch }}{{ end }}
```

Comment:
note: I'm surprised `.Arch` would match `darwin` I thought `.Os` would be `darwin` and `.Arch` would be `aarch64`

Reply:
updated